### DISCUSSION TITLE
Web app?

### DISCUSSION THREAD

**Discussion by ghost** [Kong/insomnia#6213] [Ideas]
Have you guys considered making Insomnia available as a web app? Would love to hear your thoughts on this.

**Comment by cmarmonier** [comment]
What is the point?
I migrated Postman to Insomnia because it is impossible to use Postman offline: important for me for testing local webapps in a closed environment.

**Comment by ghost** [comment]
To clarify - I'm not suggesting Insomnia to be available only as a web app but also as a web app. This will let new users try out Insomnia without having to commit to downloading an app. For local testing an offline app will always be better.
